core/src/it/java/org/seedstack/business/assembler/auto/fixture/DummyDefaultAssembler.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2013-2015 by The SeedStack authors. All rights reserved.
+ *
+ * This file is part of SeedStack, An enterprise-oriented full development stack.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.seedstack.business.assembler.auto.fixture;
+
+import com.google.inject.assistedinject.Assisted;
+import org.seedstack.business.api.DefaultImpl;
+import org.seedstack.business.api.domain.AggregateRoot;
+import org.seedstack.business.api.interfaces.assembler.AbstractBaseAssembler;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import java.lang.reflect.Method;
+
+/**
+ * @author pierre.thirouin@ext.mpsa.com (Pierre Thirouin)
+ */
+@DefaultImpl
+@Named("Dummy")
+public class DummyDefaultAssembler<A extends AggregateRoot<?>,D> extends AbstractBaseAssembler<A, D> {
+
+
+    @SuppressWarnings("unchecked")
+    @Inject
+    public DummyDefaultAssembler(@Assisted Object[] genericClasses) {
```

Comment:
Do projects have to use `@Assisted` which is a guice specific class or this is just for the tests ?